## Problem Statement - Normal Distribution

The mean weight of a morning breakfast cereal pack is 0.295 kg with a standard deviation of 0.025 kg. The random variable weight of the pack follows a normal distribution. 

  a)What is the probability that the pack weighs less than 0.280 kg?
  
  b)What is the probability that the pack weighs more than 0.350 kg? 
  
  c)What is the probability that the pack weighs between 0.260 kg to 0.340 kg? 

In [14]:
import scipy.stats as stats

In [15]:
# Z-Score for x=0.280 for mean of 0.295 and sdev of 0.025
# z = (x-mean)/sdev

z=(0.28-0.295)/0.025

In [16]:
z

-0.5999999999999983

- we need to find the area of curve to the left of z value, for a z value of -0.6, which will be the probability that the pack weighs less than 0.280 kg

In [17]:
#cdf is cumulative density function (continuous area) which gives the area under the curve for z value of -0.6

stats.norm.cdf(-0.6)

0.2742531177500736

- A cdf value of 0.274 tells that there is a 27.4% chance of the cereal pack weighing less than 280 grams, when it is randomly choosen from the cereal packs

In [18]:
# Z-Score for x=0.350 for mean of 0.295 and sdev of 0.025
# z = (x-mean)/sdev

z=(0.35-0.295)/0.025

In [19]:
z

2.1999999999999997

- we need to find the area of curve to the right of z value, for a z value of 2.2, which will be the probability that the pack weighs more than 0.350 kg

In [20]:
1 - stats.norm.cdf(2.2)

0.01390344751349859

- A cdf value of 0.013 tells that there is a 1.4% chance of the cereal pack weighing more than 350 grams , when it is randomly choosen from the cereal packs

In [21]:
# Z-Score for x=0.260 for mean of 0.295 and sdev of 0.025
# z = (x-mean)/sdev

z1= (0.26-0.295)/ 0.025

In [22]:
# Z-Score for x=0.340 for mean of 0.295 and sdev of 0.025
# z = (x-mean)/sdev

z2 = (0.34-0.295)/0.025

In [23]:
z1, z2

(-1.399999999999999, 1.8000000000000016)

- we need to find the difference in the area of curve to the right of z1=1.8  and the area of curve to the left of z2=-1.4 value, which will be the probability that the weight of the pack selected at random is between 0.260 kg and 0.340 kg

In [24]:
stats.norm.cdf(z2) - stats.norm.cdf(z1)

0.8833130216533032

- A cdf value of 0.88 tells that there is an 88% chance of the cereal pack weighing more than 260 grams and less than 340 grams, when it is randomly choosen from the cereal packs